docs/docs-components/SandpackExample.js

```diff
@@ -112,17 +97,15 @@ export default function SandpackExample({
   layout = 'row',
   name,
   previewHeight,
+  showControls = true,
   showEditor = true,
-  hideControls = false,
-  mobileView = false,
 }: {|
   code: ?string | (() => Node),
-  layout?: 'row' | 'column',
+  layout?: 'row' | 'column' | 'mobileRow',
   name: string,
-  previewHeight?: 'sm' | 'md' | number,
+  previewHeight?: number,
+  showControls?: boolean,
   showEditor?: boolean,
-  hideControls?: boolean,
-  mobileView?: boolean,
```

Comment:
@AlbertCarreras fyi I merged `mobileView` into the `layout` prop with `mobileRow`. It would be great if we could limit the amount of props available on `SandpackExample`